src/components/Event/List/List.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import EventItem from '../Item/Item';
+
```

Comment:
not sure if this empty line is here on purpose?

Reply:
Yes, empty line was added because in my opinion it is cleaner when packages, components, styles and screens imports are seperate by the empty line

Reply:
👍